src/test/regress/sql/failure_vacuum.sql

```diff
@@ -0,0 +1,43 @@
+-- print whether we're using version > 10 to make version-specific tests clear
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+')::int > 10 AS version_above_ten;
+
+SELECT citus.mitmproxy('conn.allow()');
+
+SET citus.shard_count = 1;
+SET citus.shard_replication_factor = 2; -- one shard per worker
+SET citus.multi_shard_commit_protocol TO '1pc';
+
+CREATE TABLE vacuum_test (key int, value int);
+SELECT create_distributed_table('vacuum_test', 'key');
+
+SELECT citus.clear_network_traffic();
+
+SELECT citus.mitmproxy('conn.onQuery(query="^VACUUM").kill()');
+VACUUM vacuum_test;
+
+SELECT citus.mitmproxy('conn.onQuery(query="^ANALYZE").kill()');
+ANALYZE vacuum_test;
+
+SELECT citus.mitmproxy('conn.onQuery(query="^COMMIT").kill()');
+ANALYZE vacuum_test;
+
+-- TODO: This feels unnecessary; why are failed ANALYZE transactions critical?
```

Comment:
interesting find, would it be worth to look into this in a future? In that case an issue on github might be more visible then a TODO.

Reply:
Agreed; I'll open an issue.